source/user-manual/capabilities/vulnerability-detection/offline-update.rst

```diff
@@ -265,7 +265,7 @@ Alternatively, the feeds can be loaded from a local path. To achieve it, configu
 
     <provider name="arch">
         <enabled>yes</enabled>
-        <path>/local_path/archlinux_all.json$</path>
+        <path>/local_path/archlinux_all\.json$</path>
```

Comment:
This is a single file and we don't need a regex.
But we could change `archlinux_all.json` for `all.json` because that's the name of the file the documentation suggests to download.

Reply:
Escaping POSIX regex special characters to meet literals is the right way to fix this. A note will be appended to current documentation to make this clear.
`/local_path/all\.json` approach is the better one